snuba/subscriptions/executor_consumer.py

```diff
@@ -46,6 +47,29 @@
 COMMIT_FREQUENCY_SEC = 1
 
 
+def calculate_max_concurrent_queries(
+    curr_partition_count: int,
```

Comment:
suggestion `assigned_partition_count` - curr is a bit ambiguous about how it differs from total